translations/translations.json

```
@@ -268,12 +268,14 @@
   },
   "chat": {
     "de": {
+      "botIcon": "Chatbot",
```

:wrench: Generally we try to use naming to reflect functionality over usage. In this case, the `Icon` is only usage related, which might frequently change, e.g. since we use it in other places than just an icon or don't use it for icons at all anymore. In those cases, we'd have to rename this every time.
```suggestion
      "bot": "Chatbot",
```